colonel/simulator/simulator.py

```diff
@@ -67,6 +87,21 @@ def parse_main_log_file(cls, file_path):
                                                       cls.MODEL_DEST_COL])
         return parsed_logs
 
+    def plot_port(self, port: Port, index=0):
```

Comment:
Missing plot return

Reply:
should I create a figure or something link that so users can use it as they wish?